src/app/public/modules/wait/wait-adapter.service.ts

```diff
@@ -17,9 +17,11 @@ export class SkyWaitAdapterService {
     this.renderer.setElementStyle(waitEl.nativeElement.parentElement, 'position', undefined);
   }
 
-  public setBusyState(waitEl: ElementRef, isFullPage: boolean, isWaiting: boolean) {
+  public setBusyState(waitEl: ElementRef, isFullPage: boolean, isWaiting: boolean, isBlocking: boolean) {
```

Comment:
Since this is a public method, any new parameters will need to be optional.
```
public setBusyState(
  waitEl: ElementRef,
  isFullPage: boolean,
  isWaiting: boolean,
  isBlocking = false // or, whatever the default should be
) {
```